intents/user/parse_student_info.py

```diff
@@ -0,0 +1,22 @@
+import json
+import os
+
+with open('intentInitStudentInfoRO.json') as intentSRLeaderRO:
+    intentRO = json.load(intentSRLeaderRO)
+with open('intentInitStudentInfoEN.json') as intentSRLeaderEN:
+    intentEN = json.load(intentSRLeaderEN)
+with open('../../../resources/nrSR.json') as nrSRModel:
+    nrSR = json.load(nrSRModel)
+f = open("generatedIntentSRLeaderRO.yml", "w")
+fEN = open("generatedIntentSRLeaderEN.yml", "w")
```

Comment:
You generate intentions for initStudentInfo, but still the files generated by you are for SRLeader intention. Change the names.